js/app.js

```diff
@@ -0,0 +1,483 @@
+// console.log('Homework!!')
+
+// Q & A
+
+// 1. We assign a variable by declaring it within the js document.
+// 	  You can declare a variable with var, let, or const. Each one 
+// 	  has its uses except var... never use var... you can also declare
+//	  the variable without any of these, but that's like declaring it
+//	  var. Don't do it.
```

Comment:
it's _even worse_ than using var if you can believe that
also there are times when var is sensible, so don't think of it as poisonous